### PR TITLE
Weekly automation test

### PR DESCRIPTION
**Pull request by ghost**
This change is NOT REVIEWABLE

---
This change is [Reviewable](https://reviewable.io/reviews/fabianorosas/builds/1)

### CODE REVIEW

**Comment by ghost** [comment]
Review status: 0 of 17 files reviewed at latest revision, 2 unresolved discussions.

---

_[config.yaml, line 7 at r4](https://reviewable.io:443/reviews/fabianorosas/builds/1#-KWDmD0zs3WWqT_qm4Q4:-KWDmD0zs3WWqT_qm4Q5:b-tvcrxx) ([raw file](https://github.com/fabianorosas/builds/blob/07431384dd1dcd7b7b212f649a1d1e4811d8b8c4/config.yaml#L7)):_

> ``` YAML
>  build_versions_repository_url: "https://github.com/open-power-host-os/versions.git"
>  build_version: 'master'
>  distro_name: 'centos'
> ```

Testing comments. _italic_

---

_[create_release_notes.py, line 1 at r4](https://reviewable.io:443/reviews/fabianorosas/builds/1#-KWDmSq6_Iq3NZ-QZCm0:-KWDmSq7n3xpyApPSa85:bb74njr) ([raw file](https://github.com/fabianorosas/builds/blob/07431384dd1dcd7b7b212f649a1d1e4811d8b8c4/create_release_notes.py#L1)):_

> ``` Python
> # Copyright (C) IBM Corp. 2016.
> ```

Test

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/fabianorosas/builds/1)_

**Comment by farosas** [comment]
_[config.yaml, line 5 at r4](https://reviewable.io:443/reviews/fabianorosas/builds/1#-KWDnRMoN3WsR55A9rJP:-KWDnRMpaM0lU3UApXQw:b-ttmk9z) ([raw file](https://github.com/fabianorosas/builds/blob/07431384dd1dcd7b7b212f649a1d1e4811d8b8c4/config.yaml#L5)):_

> ``` YAML
>  log_file: "/var/log/host-os/builds.log"
>  repositories_path: "/var/lib/host-os/repositories"
>  build_versions_repository_url: "https://github.com/open-power-host-os/versions.git"
> ```

what is this for?

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/fabianorosas/builds/1)_

**Comment by ghost** [comment]
commenting in github

**Comment by farosas** [comment]
_[create_release_notes.py, line 1 at r4](https://reviewable.io:443/reviews/fabianorosas/builds/1#-KWDmSq6_Iq3NZ-QZCm0:-KWDnuEDVnv7TRKqbnHR:b-pb9bue) ([raw file](https://github.com/fabianorosas/builds/blob/07431384dd1dcd7b7b212f649a1d1e4811d8b8c4/create_release_notes.py#L1)):_
<details><summary>

<i>Previously, olavphibm (Olav Philipp Henschel) wrote…</i></summary>



> Test
> </details>
> indeed

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/fabianorosas/builds/1)_

File: host-os-build.py [hunk 1]
```diff
@@ -40,15 +40,16 @@ def main(args):
         CONF.get('default').get('arch_and_endianness'))
 
     LOG.info("Building packages: %s", ", ".join(packages_to_build))
-    build_manager = manager.BuildManager(packages_to_build, distro)
+    bm = build_manager.BuildManager(packages_to_build, distro)
     try:
-        build_manager()
+        bm()
```
Comment (ghost) [inline host-os-build.py:43]: Commenting inline in github

**Comment by farosas** [comment]
:lgtm:

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/fabianorosas/builds/1#-:-KWDonVRjFINhy5IhraO:bnfp4nl)_

File: lib/repository.py [hunk 2]
```diff
@@ -123,10 +123,8 @@ def _update_submodules(self):
         """
         Update repository submodules, initializing them if needed.
         """
-        # TODO(olavph): use git.objects.submodule.base.Submodule instead
-        # of run_command
-        cmd = "git submodule init; git submodule update"
-        utils.run_command(cmd, cwd=self.working_tree_dir)
+        for submodule in self.submodules:
+            submodule.update(init=True)
```
Comment (ghost) [inline lib/repository.py:127]: commenting inline in github

**Pull request closed without merging**